.github/actions/spelling/expect.txt

```diff
@@ -25,6 +26,7 @@ AType
 AUrl
 Aysnc
 azurewebsites
+baced
```

Comment:
This isn't necessary, but it's ok, it'll get cleaned out eventually:
https://github.com/microsoft/winget-cli/runs/2783873246#step:3:61